src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = {
```

Comment:
Use an es6 export instead of commonjs, so:

```js
export function list(settings, logger) {
```

instead of `module.exports = {...}`.